### ISSUE TITLE
How to initially display a maximizable dialogue at original size

### ISSUE THREAD

**Issue by dkontod** [volkanceylan/Serenity#1772]
Hello.
I used the new generation option "Create maximizable dialogs". This had as result the generation of dialogs with a "maximize" button at the upper right of the form, but it initially displays maximized. 
It does this adding at the xyzDialog.ts the line:
@Serenity.Decorators.maximizable()
How can I make it to initially display at the original size and maximize it when I like to?

**Comment by Estrusco** [comment]
If you use the search issue feature of github you get response. This question is posted and resolved two hours ago :\

**Comment by dkontod** [comment]
Thank you @Estrusco . I suppose you are referring to issue #1766. This has a misleading title, not compatible with my question. It took really too much time to find out the answer in there. Finally I assumed (and I was right judging from the result) that I had to comment out the generated code which clicks the maximize button. 
So I write here the work around I have found for reference to other that have the same question as me:
Open xyzDialog.ts
Find and comment or delete the following code:
dialogOpen() {
           super.dialogOpen();
           this.element.closest(".ui-dialog").find(".ui-icon-maximize-window").click();
}
